engines/python/setup/djl_python/rolling_batch/vllm_rolling_batch.py

```diff
@@ -35,32 +38,35 @@ def __init__(self, model_id_or_path: str, properties: dict,
         :param model_id_or_path: Currently unused since there is a copy inside properties
         :param properties: other properties of the model, such as decoder strategy
         """
-        engine_config = VllmRbProperties(**properties)
-        super().__init__(engine_config, kwargs.get("model_config", None))
-        self.init_engine()
-
-    def init_engine(self):
-        """
-        Initializes vllm engine
-        """
+        self.vllm_configs = VllmRbProperties(**properties)
+        super().__init__(waiting_steps=self.vllm_configs.waiting_steps,
+                         output_formatter=self.vllm_configs.output_formatter)
+        self.supports_speculative_decoding = supports_speculative_decoding()
+        engine_kwargs = {}
+        if supports_speculative_decoding():
```

Comment:
this change is added to not throw error if OSS vllm is used in the container, based on feedback from https://github.com/deepjavalibrary/djl-serving/pull/1595